test/recipes/70-test_sslsigalgs.t

```diff
@@ -114,7 +126,7 @@ SKIP: {
     #Test 8: Sending a valid sig algs list but not including a sig type that
     #        matches the certificate should fail in TLSv1.3.
     $proxy->clear();
-    $proxy->clientflags("-sigalgs ECDSA+SHA256");
+    $proxy->clientflags("-sigalgs ".randcase("ECDSA+SHA256"));
```

Comment:
OK, I kind of get it now by reading this -- it _is_ the second interpretation.

Reply:
I've pushed a fixup. More clear now?